packages/jest-preset-default/jest-preset.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
 		'**/?(*.)test.[jt]s?(x)',
 	],
 	testPathIgnorePatterns: [ '/node_modules/', '<rootDir>/vendor/' ],
-	timers: 'fake',
+	timers: 'real',
```

Comment:
This looks like it will need a CHANGELOG entry. For some relying on fake timers to be the default, this might be a breaking change.

Reply:
I'll return to this in a followup. Now my goal is to:
- land this real timers change
- make `useSelect` with `useSyncExternalStore` pass all the tests and merge it
- try out the "enable concurrent mode" PR with the new `useSelect`, to see if the current e2e failures disappear or not